### DISCUSSION TITLE
Get status from unit

### DISCUSSION THREAD

**Discussion by mongojarle** [bisand/node-red-contrib-panasonic-comfort-cloud#6] [Q&A] [answered]
Hello, looks great! Looking for the best heatpump to buy, and came across your contribution.

Will your nodes also get the current status of the hest pump? So if I change the settings on the pump with the remote, I can get the update setting in my flow?

**Comment by bisand** [comment, marked as answer]
Thanks! 

Yes, the nodes will be able to get the current status if you change parameters via remote or app, but you will have to pull the data, they are not pushed. Ofcourse it requires the heatpump to be connected to internet and you have an account on [Panasonic Comfort Cloud](https://aircon.panasonic.com/connectivity/business/comfort_cloud.html)

**Reply by mongojarle** [reply]
Thank you very much! Sees like a Panasonic pump is what I need for my smarthome. Excelent work!
